storage/types.go

```diff
@@ -33,19 +33,61 @@ type Node struct {
 }
 
 // NodeID uniquely identifies a Node within a versioned MerkleTree.
+// NodeIDs as presented to the storage layer will always have a multiple
+// of 8 bits as they identify the root of a 256 element subtree.
+//
+// Reading paths right to left is the natural order when traversing from
+// leaves towards the root. However, for internal nodes the rightmost bits
+// of the IDs are not aligned on a byte boundary so care must be taken.
+//
+// Note that some of the APIs count bits with 0 being the rightmost
+// in the entire path array when some of these might not be significant.
 type NodeID struct {
-	// path is effectively a BigEndian bit set, with path[0] being the MSB
+	// Path is effectively a BigEndian bit set, with Path[0] being the MSB
```

Comment:
"B" in MSB is used both for bits and bytes throughout this type. Expand one of them to avoid ambiguity?

Reply:
Fixed so MSB is always referring to bits.